credentials/tls/certprovider/store.go

```diff
@@ -0,0 +1,126 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package certprovider
+
+import (
+	"sync"
+
+	"google.golang.org/grpc/grpclog"
+)
+
+// store is the global singleton certificate provider store.
+var store *Store
+
+func init() {
+	store = &Store{
```

Comment:
Do you need this to be in `init()`?
Does `var store = &Store{` work?

Reply:
Done.